Cargo.toml

```
@@ -137,6 +136,9 @@ hashbrown = { version = "0.12.0", features = [
 ] }
 dashmap = "5.3"
 
+[target.'cfg(linux)'.dependencies]
```

```suggestion
[target.'cfg(target_os = "linux")'.dependencies]
```

https://doc.rust-lang.org/reference/conditional-compilation.html